examples/filepath/tests/filepath_test.bal

```diff
@@ -0,0 +1,61 @@
+import ballerina/io;
+import ballerina/test;
+import ballerina/system;
+
+any[] outputs = [];
+int counter = 0;
+
+// This is the mock function which will replace the real function
+@test:Mock {
+    moduleName: "ballerina/io",
+    functionName: "println"
+}
+public function mockPrint(any... s) {
+    outputs[counter] = s[0];
+    counter += 1;
+}
+
+boolean IS_WINDOWS = system:getEnv("OS") != "";
```

Comment:
Why do we have this field in capital if it's not a constant? also if it is only used in this file we can make it private. private fields are now supported.